phys/module_cu_tiedtke.F

```diff
@@ -2936,6 +2936,9 @@ subroutine cuentr_new                              &
             ztmzk = -(pgeoh(jl,ikh)-pgeoh(jl,ikt))*zrg
             arg = 3.1415*(zzmzk/ztmzk)*0.5
             zorgde = tan(arg)*3.1415*0.5/ztmzk
+#ifdef aarch64_x86_correctness_fix
+            zorgde = real(tan(dble(arg))*3.1415*0.5/dble(ztmzk))
+#endif
```

Comment:
With the preprocessing directives here and in `module_sf_myjsfc.F` as written now, it looks like some calculations will be done twice when `aarch64_x86_correctness_fix` is defined; is that intended?

Reply:
Yes, #else could be used, but if this is just for testing and not full runs, the efficiency won't matter.

Reply:
We do intend this to be used for future runs on arm64 machines for customers who have concerns over correctness. I have updated the PR as suggested.  Thanks,